apps/swiss/sh/util.c

```diff
@@ -1112,7 +1112,7 @@ Return Value:
 
 {
 
-    CHAR Character;
```

Comment:
I think the type of this should stay as CHAR, as it matches PSTR where elements come from, and makes the comparison to things like SHELL_CONTROL_ESCAPE make more sense.

Instead, can you find the place where isspace is used, and change it to something like isspace((UCHAR)Character), maybe with a comment explaining that these functions must take a value representable as an unsigned char. Looks like POSIX has similar language to MSDN:

http://pubs.opengroup.org/onlinepubs/000095399/functions/isspace.html

I wonder if there are other places where we're calling this family of functions with these odd values.